### PR TITLE
wolfMQTT Release v1.15.1 preparation

### PR DESCRIPTION
**Pull request by embhorn**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Release Notes
 
+### v1.15.1 (3/14/2023)
+* Update to release send lock after the compare of write.len #318
+* Zd15712 correct send unlocking #2 #319
```
Comment (dgarske) [inline ChangeLog.md:5]: Would prefer to this be combined with the above line. No ZD number or `#2`. Something like: 

Fix to protect read from `client->write.len`, which resolve edge case issue when using multi-threading. (PR #318 and PR #319).

**Review by dgarske — DISMISSED** [review]

**Comment by embhorn** [comment]
@billphipps - @dgarske has already done a preliminary review. Please review and merge this PR.

File: IDE/F767ZI-TOPPERS/user_settings.h [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /* user_settings.h
  *
- * Copyright (C) 2006-2022 wolfSSL Inc.
+ * Copyright (C) 2006-2023 wolfSSL Inc.
```
Comment (billphipps) [inline IDE/F767ZI-TOPPERS/user_settings.h:3]: Do we need to update copyright on files with no changes?  Just looking for policy.

**Review by billphipps — APPROVED** [review]

File: IDE/F767ZI-TOPPERS/user_settings.h (hunk 2 above)
Comment (embhorn) [reply IDE/F767ZI-TOPPERS/user_settings.h:3]: Yes

**Pull request merged**